scalyr_agent/builtin_monitors/docker_monitor.py

```diff
@@ -1234,6 +1245,120 @@ class DockerMonitor( ScalyrMonitor ):
     It can also collect all log messages written to stdout and stderr by those containers, in conjunction with syslog.
     See the online documentation for more details.
 
+    ## Docker Labels
+
+    You can get the Scalyr agent to upload custom attributes for your containers by setting labels on the container, and then configuring the agent to upload labels as attributes.  This is controlled by a number of config options:
```

Comment:
`You can configure the Scalyr Agent to upload customer attributes for your containers based on the labels you set on the container itself.  This can be used to easily set the parser that should be used to parse the container's log, as well as adding in arbitrary labels on the container's log.

To use this functionality, you must properly configure the agent by setting the `labels_as_attributes` configuration option to `true`.  All of the docker monitor configuration options related to this feature are as follows:`